### PR TITLE
Full page 1st translation

### PR DESCRIPTION
**Pull request by koyakei**

### CODE REVIEW

File: ja/orm/entities.rst [hunk 1]
```diff
@@ -1,17 +1,486 @@
...
+エンティティーはそれらのデータにアクセスして操作するための継続的なプロパティーとメソッドを含みます。
+
+エンティティーはCakePHPによって ``find()`` を使った時に毎回テーブルオブジェクトに作られます。
+
+エンティティークラスの生成
+=======================
+
+CakeのORMを使うためにエンティティークラスを生成する必要はありません。
+でも、使用するエンティティでロジックをカスタマイズしたいなら、クラスを作る必要があります。
+規約に従うとと **src/Model/Entity/** にクラスが存在するはずです。
```
Comment (suzuki) [inline ja/orm/entities.rst]: **とと**

**Comment by suzuki** [comment]
@koyakei fix grammer, plz.

**Comment by koyakei** [comment]
fixed

**Comment by suzuki** [comment]
@koyakei thx !

**Pull request merged**